package.json

```diff
@@ -46,7 +46,7 @@
         "axios": "^0.21.1",
         "debug": "~2.6.9",
         "lodash": "^4.17.15",
-        "moment": "^2.29.2",
+        "luxon": "^3.2.1",

```

Comment:
Unless there's a compelling reason to use Luxon, I think we should use dayjs. It's way more popular than Luxon and it specifically calls itself an "alternative to Moment.js"

https://npmtrends.com/dayjs-vs-luxon

Reply:
I'm not sure why `Luxon` was picked as the alternative instead of `dayjs`. @davidmrdavid thoughts?

I will say, `Luxon` was also created by the creators of `moment.js`, and using `Luxon` for this migration, I've liked how more strict and type-safe it feels compared to `moment.js`, something that `dayjs` doesn't do from what I've seen.

Reply:
I think `Luxon` was suggested by a third party user who notified us of `moment.js` now longer being maintained. I have no strong feelings about the use of library.